src/QMCHamiltonians/SkAllEstimator.cpp

```diff
@@ -105,7 +104,7 @@ void SkAllEstimator::evaluateIonIon()
 
 SkAllEstimator::Return_t SkAllEstimator::evaluate(ParticleSet& P)
 {
-  RealType w = tWalker->Weight;
+  RealType w = tWalker->Weight; // Segfaults unless setHistories() is called prior
```

Comment:
Could you move the comment above this line?

Reply:
Fixed.